lib/validate.js

```diff
@@ -191,7 +191,10 @@ module.exports = {
       if (_.has(this.serverless, 'service.package') && this.serverless.service.package.individually) {
         this.multiCompile = true;
         this.serializedCompile = this.configuration.serializedCompile;
-        this.options.verbose && this.serverless.cli.log(`Using ${this.serializedCompile ? 'serialized' : 'multi'}-compile (individual packaging)`);
+        this.options.verbose &&
+          this.serverless.cli.log(
+            `Using ${this.serializedCompile ? 'serialized' : 'multi'}-compile (individual packaging)`
+          );
```

Comment:
Was it required by the linter?

Reply:
Yes, in fact it made that change for me when I was trying to check it in. Must be a pre-commit hook or something.